src/redux/auth/reducer.js

```diff
@@ -0,0 +1,19 @@
+import { actionTypes } from './actions';
+
+const initialState = {
+  loginError: null,
+  signUpError: null
+};
+
+const authReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case actionTypes.LOG_IN_ERROR:
```

Comment:
if sign in is success you should clear the `loginError` and `signUpError` states